### PR TITLE
Adds audit triggers to moped_proj_personnel and moped_proj_personnel_roles tables

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/16149

## Testing
**URL to test:** 

local

**Steps to test:**
- run the sql command below and note the values for the audit fields (`created_by_user_id`, `created_at`, `updated_by_user_id`, and `updated_at`) for the project, personnel and personnel roles tables
- add a new team member to a project
- `updated_by_user_id` and `updated_at` should have updated for all records on the project table, and all audit fields should have updated for the new record you created on the personnel and personnel roles tables
- edit the new record's notes field and confirm that the `updated_at` and `updated_by_user_id` fields have been updated for the project and personnel tables
- go to the entry you created and updated its role by unchecking one and checking a different one
- `updated_by_user_id` and `updated_at` should have updated for all records on the project table, `updated_at` and `updated_by_user_id` should be updated for both new records on the personnel table, and all audit fields should be updated for the new record in the personnel roles table
- log out of moped and log back in using a different account
- go to the entry you created and updated its role by unchecking one and checking a different one
- `updated_by_user_id` and `updated_at` should have updated on the personnel roles records, along with the parent personnel records and the project table records

Query for moped_proj_personnel and moped_proj_personnel_roles
```sql
SELECT
	mp.project_id,
	mp.updated_by_user_id,
	mp.updated_at,
	mpp.project_personnel_id,
	mpp.created_at,
	mpp.updated_at,
	mpp.created_by_user_id,
	mpp.updated_by_user_id,
	mppr.project_role_id,
	mppr.created_at,
	mppr.updated_at,
	mppr.created_by_user_id,
	mppr.updated_by_user_id,
	mppr.is_deleted
FROM
	moped_project mp
	LEFT JOIN moped_proj_personnel mpp ON mp.project_id = mpp.project_id
	LEFT JOIN moped_proj_personnel_roles mppr ON mpp.project_personnel_id = mppr.project_personnel_id
WHERE
	mp.project_id = <your project id>;
```

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Review by chiaberry — APPROVED** [review]
test steps check out!

Unrelated, I noticed that when I update a personnel's role, the activity log says I updated the notes. I'll open an issue

File: moped-database/seeds/1602292389297_initial_seed_staging.sql
Comment (mddilley) [inline moped-database/seeds/1602292389297_initial_seed_staging.sql:1]: Nice catch! It is so easy to forget the seed data. 🙏

**Review by mddilley — APPROVED** [review]
This works so well! I tested the migrations locally, and all of your steps with multiple users. Reviewing this gave me a whole new appreciation for the changes needed throughout the code to get these values to propagate up to the parent project. Thanks for sharing as you worked through this too! 🚢 🚀 🚢

File: moped-database/migrations/1713213493447_add_personnel_personnel_role_audit_fields/down.sql
[the remark this answers is not in the document]
Comment (chiaberry) [reply moped-database/migrations/1713213493447_add_personnel_personnel_role_audit_fields/down.sql:1]: I think this is a good idea, our readmes could certainly use some general editing and updating as well.

[the remark this answers is not in the document]
Comment (tillyw) [reply moped-database/migrations/1713213493447_add_personnel_personnel_role_audit_fields/down.sql:1]: thanks for pointing this out @mddilley! I'll make sure to set up the autoformatting for future PRs!

**Comment by chiaberry** [comment]
merging so we can begin testing

**Pull request merged**